cvat-ui/src/actions/annotation-actions.ts

```diff
@@ -1535,3 +1540,51 @@ export function redrawShapeAsync(): ThunkAction {
         }
     };
 }
+
+export function trackerSettings(name: string, value: string | number): AnyAction {
+    return {
+        type: AnnotationActionTypes.TRACKER_SETTINGS,
+        payload: {
+            name,
+            value,
+        },
+    };
+}
+
+export function resetTrackerSettings(): AnyAction {
+    return {
+        type: AnnotationActionTypes.RESET_TRACKER_SETTINGS,
+    };
+}
+
+export function doTracking(body: any, taskId: string): ThunkAction {
```

Comment:
Why do not use type ``TrackerPayload`` instead of ``any``?
Also let's change type of ``taskId`` to a number instead of ``string``

Reply:
Yes. Changed.